pkg/skaffold/deploy/helm.go

```diff
@@ -554,18 +536,34 @@ func installArgs(r latest.HelmRelease, builds []build.Artifact, valuesSet map[st
 		args = append(args, "--set-string", value)
 	}
 
+	args, err = constructOverrideArgs(&r, builds, args, func(k string) {
```

Comment:
@tstromberg currently `skaffold render` for helm passes empty values for `image.name` and `image.tag`. Also it skips over `--set-file`. Shouldn't `deploy` and `render` lead to identical kubernetes manifest files? That's what I assumed and extracted the common `constructOverrideArgs` method. Let me know if that shouldn't be the case.

Reply:
It seems logical to me that deploy and render should yield identical manifests, but there are likely corner cases to be considered. @nkubala - thoughts?

Reply:
there are some upcoming features to `render` that will shift the paradigms a bit from the conventional `deploy` flow, but for the most part they should be going through the same code paths so this is fine.